Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,780 @@
+installation location.  For this example, we will utilize `/opt/sst/elements` as our 
+installation target.  At this point, we can configure, build and install 
+the SST-Elements infrastructure as follows:
+```
+$> mkdir /opt/sst/elements
+$> ./configure --prefix=/opt/sst/elements --with-sst-core=/opt/sst
+$> make -j
+$> make install
+$> sst-info -q
+```
+
+At this point in the tutorial, we have a functional SST environment 
+with both the base SST-Core and the SST-Elements packages installed.
+
+## Basic Commands <a name="BasicCommands"></a>
+
+There are a number of basic commands that are commonly utilized when 
+running and/or debugging an SST simulation.  We summarize the most 
+commonly used command sets here and the situations where they are 
+appropriately utilized.  However, all of the major SST command line tools 
+provide a `--help` option to print useful command line options 
+and their associated parameters.  
+
+### SST Execution and Utility Functions
+
+* _Executing a Simulation_: The most common command line tool is the `sst` command.  
+This is utilized to build and execute a simulation run using the SST Core and associated 
+Elements libraries.  Using the `sst` command line tool and passing a loadable 
+simulation graph using Python or JSON can be performed as follows:
+```
+$> sst /path/to/simulation.py
+$> sst /path/to/simulation.json
+```
+
+* _Executing with Verbosity_: Similar to the command above, users may enable 
+verbose output of the graph construction and simulation by passing the `-v`
+or `--verbose=LEVEL` option.  The higher the `LEVEL` specified, the more verbose
+the output will be.  This is often useful for debugging new simulation configurations.  
+```
+$> sst -v /path/to/simulation.py
+$> sst --verbose=4 /path/to/simulation.py
+```
+
+* _Testing Simulation Graph Syntax_: One of the common issues that users encounter 
+when constructing complex simulation graphs is basic syntactic issues in their 
+respective input files that prevent SST from constructing a valid simulation.  It is 
+often useful to utilize the SST graph loader(s) to validate a graph using the 
+internal initialization functions without actually executing the simulation.  
+This can be performed using the following command.  Note that SST will report 
+that the simulation completed, but the reported simulation time will be 0s.
+```
+$> sst --run-mode=init /path/to/simulation.py
+Simulation is complete, simulated time: 0 s
+```
+
+* _Outputting DOT Configuration Graphs_: It is also often useful for debugging 
+or publication purposes to direct SST to output a GraphViz DOT file that contains 
+the configuration graph of the candidate simulation.  This can be performed 
+with or without actually executing the simulation.  We show examples for both 
+as follows:
+```
+$> sst --output-dot=foo.dot /path/to/simulation.py
+$> sst --run-mode=init --output-dot=foo.dot /path/to/simulation.dot
+$> dot -Tpdf foo.dot > foo.pdf
+```
+
```

Comment:
Please include the `dot-verbosity` documentation; see https://github.com/researcherben/structural-simulation-toolkit/issues/25

Reply:
Added additional sample runtime command in a48b6c9.  Per the current code in the `master` branch, any values < 10 will only print the configuration graph.  Values >= 10 will print the configuration graph partitioned by MPI rank.